### DISCUSSION TITLE
How do I add contains to or?

### DISCUSSION THREAD

**Discussion by ru88s** [supabase/supabase#2043] [Questions] [answered]
```
const { data, error, count } = await supabase
      .from("talks")
      .select("*", { count: "exact" })
      .or(`title.ilike.%${query}%,tags.contains.["${query}"]`);
```

Can I use contains for or?
Is there any way to get it this way?

**Comment by soedirgo** [comment, marked as answer]
Hey @ru88s, for contains it should be `tags.cs.{${query}}` (curly braces for arrays and no quotes). You can see [here](https://postgrest.org/en/stable/api.html#horizontal-filtering-rows) to get an idea of the `or` syntax.

**Reply by ru88s** [reply]
It's resolved!
Thank you for the quick answer.
